tests/integration/test_model.py

```diff
@@ -47,140 +47,176 @@ def check_trained_model(c, model_name=None):
 @pytest.fixture()
 def training_df(c):
     df = timeseries(freq="1d").reset_index(drop=True)
-    c.create_table("timeseries", df, persist=True)
-
-    return None
-
-
-@pytest.fixture()
-def gpu_training_df(c):
     if dask_cudf:
-        df = timeseries(freq="1d").reset_index(drop=True)
         df = dask_cudf.from_dask_dataframe(df)
         c.create_table("timeseries", input_table=df)
+    else:
+        c.create_table("timeseries", df, persist=True)
```

Comment:
Think we'll want the CPU/GPU tables to have separate names that we parametrize on in tests, otherwise we end up defaulting to GPU testing whenever dask-cudf is importable